src/noui.h

```diff
@@ -17,4 +19,16 @@ void noui_InitMessage(const std::string& message);
 /** Connect all bitcoind signal handlers */
 void noui_connect();
 
+/** Suppress all bitcoind signal handlers. Used to suppress output during test runs that produce expected errors */
+void noui_suppress();
+
+/** Non-GUI handlers, which just ignore the message. Connected when calling noui_suppress.
```

Comment:
I don't think there's a need to have these three functions externally exposed in the header, they're only used in `noui_suppress`

Reply:
Adjusted